docker_utils/rtenvsub.sh

```diff
@@ -20,6 +20,7 @@ version='0.1'
 # Set a safe umask
 umask 0077
 
+DEV_MODE="${DEV_MODE:-false}"
```

Comment:
It's probably better to use a lower case variable inside the script but consume the upper case one for clarity:

dev_mode="${DEV_MODE:-false}"